qcodes/tests/test_station.py

```diff
@@ -201,3 +215,306 @@ def test_station_after_instrument_is_closed():
+    assert has_station_config_been_loaded(Station(config_file=path.name))
+
+    config["default_file"] = None
+    config["default_folder"] = None
+    assert has_station_config_been_loaded(Station(config_file=str(path)))
+
+
+def test_station_creation(example_station):
+    assert "StationConfigurator" in example_station.components.keys()
+
+SIMPLE_MOCK_CONFIG = """
+instruments:
+  mock:
+    driver: qcodes.tests.instrument_mocks
+    type: DummyInstrument
+"""
+
+@pytest.fixture
+def simple_mock_station(example_station_config):
+    yield station_from_config_str(SIMPLE_MOCK_CONFIG)
+
+def test_simple_mock_config(simple_mock_station):
+    st = simple_mock_station
+    assert has_station_config_been_loaded(st)
+    assert hasattr(st, 'load_mock')
+    mock_snapshot = st.snapshot()['components']['StationConfigurator']\
+        ['instruments']['mock']
+    assert mock_snapshot['driver'] == "qcodes.tests.instrument_mocks"
+    assert mock_snapshot['type'] == "DummyInstrument"
+
+
+def test_simple_mock_load_mock(simple_mock_station):
+    st = simple_mock_station
+    mock = st.load_mock()
+    assert isinstance(mock, DummyInstrument)
+    assert mock.name == 'mock'
+    assert st.components['mock'] is mock
+
+
+def test_simple_mock_load_instrument(simple_mock_station):
+    st = simple_mock_station
+    mock = st.load_instrument('mock')
+    assert isinstance(mock, DummyInstrument)
+    assert mock.name == 'mock'
+    assert st.components['mock'] is mock
+
+
+def test_enable_force_reconnect() -> None:
+    def get_instrument_config(enable_forced_reconnect: Union[bool, None]) -> str:
+        return f"""
+instruments:
+  mock:
+    driver: qcodes.tests.instrument_mocks
+    type: DummyInstrument
+    {f'enable_forced_reconnect: {enable_forced_reconnect}'
+        if enable_forced_reconnect is not None else ''}
+    init:
+      gates: {{"ch1", "ch2"}}
+         """
+
+    def assert_on_reconnect(user_config_val: Union[bool, None],
+                            instrument_config_val: Union[bool, None],
+                            expect_failure: bool) -> None:
+        qcodes.config["station_configurator"]\
+            ['enable_forced_reconnect'] = user_config_val
+        st = station_from_config_str(
+            get_instrument_config(instrument_config_val))
+        st.load_instrument('mock')
+        if expect_failure:
+            with pytest.raises(KeyError) as excinfo:
+                st.load_instrument('mock')
+                assert ("Another instrument has the name: mock"
+                        in str(excinfo.value))
+        else:
+            st.load_instrument('mock')
+        Instrument.close_all()
+
+    for user_config_val in [None, True, False]:
+        assert_on_reconnect(user_config_val, False, True)
+        assert_on_reconnect(user_config_val, True, False)
+
+    assert_on_reconnect(True, None, False)
+    assert_on_reconnect(False, None, True)
+    assert_on_reconnect(None, None, True)
+
```

Comment:
could you use argument names in the `assert_on_reconnect` calls in this test to improve readability? reading sequences of True/False/None is not convenient.